adminSiteServer/publicApiRouter.ts

```diff
@@ -18,3 +22,12 @@ publicApiRouter.router.get(
         }
     }
 )
+
+publicApiRouter.router.get("/health", async (req: Request, res: Response) => {
+    const sqlPromise = db.mysqlFirst(`SELECT id FROM charts LIMIT 1`)
+    const timeoutPromise = rejectAfterDelay(1500) // Wait 1.5 seconds at most
+    const result = await Promise.race([sqlPromise, timeoutPromise])
+    if (result instanceof Error)
+        res.status(500).end("Error querying the database")
+    res.status(200).end("OK")
```

Comment:
This check currently doesn't work, because `await` will throw and you'll have to use a try-catch block.

This will work instead:
```suggestion
try {
        await Promise.race([sqlPromise, timeoutPromise])
        res.status(200).end("OK")
    } catch (e) {
        res.status(500).end("Error querying the database")
        // maybe add a console.log with the error here??
    }
```

Reply:
Of course, thanks for catching that. It resulted in an error code 500 which was all I cared about but this is of course what I meant to do. Thanks!